src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/MiscFeatures.kt

```diff
@@ -481,7 +482,16 @@ class MiscFeatures {
             WorldAgeDisplay()
         }
     }
-
+    @SubscribeEvent
+    fun onRenderLivingPre(event: RenderLivingEvent.Pre<EntityLivingBase>) {
+        if (!Utils.inSkyblock) return
+        if (event.entity is EntityArmorStand) {
+            val entity = event.entity as EntityArmorStand
+            if (!entity.hasCustomName() || !entity.customNameTag.contains("Jawbus") || !isBreefing()) return
```

Comment:
this is not happening

Reply:
it could if you ticktask it

Reply:
I'm also pretty confident that this doesn't even do anything since it only changes the local variable `name`, which goes out of scope immediately after.

Reply:
Ide said it'll be used like .setCustomNameTag or something similar so i just assumed it'll work